src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,122 @@
+    id="2-decorate"
+    title="Option 2: Let the agent decorate your logs."
+  >
+
+  Already have a log forwarder you like? We've got you covered. Language agents can decorate your logs with the linking metadata needed to provide access to telemetry-in-context features.
+  
+  Simply enable log decorating in your configuration, and relaunch the agent to start decorating your logs.
+  
+  newrelic.yml:
+  ```
+  application_logging.local_decorating.enabled: true
+  ```
+
+  Environment variable:
+  ```
+  NEW_RELIC_APPLICATION_LOGGING_LOCAL_DECORATING_ENABLED=true
+  ```
+  
+  Decoration is made possible by the addition of four attributes: `entity.guid`, `hostname`, `span.id`, and `trace.id`.
+
+  We recommend this option over the old decorating formatter, `NewRelic::Agent::Logging::DecoratingFormatter`.
+
+  </Collapser>
+
+  <Collapser
+    id="3-old-formatter"
+    title="Option 3: Use the old decorating formatter."
+  >
+
+  Before language agents had the ability to forward and decorate logs, you could be enable logs in context by updating your application to use the `NewRelic::Agent::Logging::DecoratingFormatter` to send linking metadata.
+
+  This option is still supported, but is no longer encouraged. For instructions to use this approach, see [Ruby: Configure logs in context](https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-ruby/).
+
+  ![Use infrastructure agent as log forwarder](./images/infra-log-forwarder.png "Use infrastructure agent as log forwarder")
+
+  If you don't have any log forwarder enabled, the UI will prompt you to use the infrastructure agent as your log forwarder.
+  </Collapser>
+</CollapserGroup>
+
+## Secure your data [#obfuscation]
+
+Your logs may include sensitive information protected by HIPAA or other compliance protocols. By default we obfuscate number patterns that appear to be for items such as credit cards or Social Security numbers, but you may need to hash or mask additional information.
+
+For more information, see our documentation about [obfuscation expressions and rules](https://docs.newrelic.com/docs/logs/ui-data/obfuscation-ui/). You can hash or mask your log data by using the New Relic UI or by using NerdGraph, our GraphQL API.
+
+## Explore your data [#view-ui]
+
+To make the most of your logging data:
+
+* On the [APM **Summary** page](https://docs.newrelic.com/docs/apm/apm-ui-pages/monitoring/apm-summary-page-view-transaction-apdex-usage-data), click your **Web transaction time** chart to view logs associated with a specific point in time.
+* Check your app's [**Errors inbox**](https://docs.newrelic.com/docs/errors-inbox/errors-inbox) to view the logs associated with your errors.
+* Use [distributed tracing](https://docs.newrelic.com/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui) to see the logs associated with individual traces.
+* Set up [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/).
```

Comment:
I think this could use some additional logging sparkle. Maybe something like this?

```suggestion
* Set up [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) based on log output and severity.
```